pkg/distributor/push_test.go

```diff
@@ -838,7 +837,7 @@ func TestHandler_DistributorPushErrorHTTPStatus(t *testing.T) {
 		},
 		{
 			name:               "a Validation gets translated into an HTTP 400",
-			err:                distributorerror.NewValidation(originalErr),
+			err:                NewValidation(originalErr),
```

Comment:
eg. `NewValidation` doesn't make much sense now that we don't call it with `distributorerror.` prefix, but `NewValidationError` would.